packages/ui5-middleware-stringreplacer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ui5-middleware-stringreplacer",
-  "version": "0.2.5",
+  "version": "0.3.0",
```

Comment:
You don't need to update the version - this happens via conventional commits. A `feat` change will create a minor version upgrade.